### PR TITLE
#5 404

### PR DESCRIPTION
**Pull request by ignacioperezroca**
### 404 - Not Found
-**Not-found.component UI**
-**Not-found.component Styles**
-**Not found contract Routing**
-**Animations to 404**

### CODE REVIEW

File: src/assets/404-tree.svg [hunk 1]
```diff
@@ -0,0 +1,385 @@
...
+							c0.6,2.1-0.6,4.3-2.7,4.9l-16.9,4.8C957.7,835.1,957.3,835.1,956.9,835.1z"/>
+					</g>
+					<g>
+						<path class="st2" d="M919.2,847.3c-1.7,0-3.3-1.1-3.8-2.9c-0.6-2.1,0.6-4.3,2.7-4.9l15.9-4.6c2.1-0.6,4.3,0.6,4.9,2.7
+							c0.6,2.1-0.6,4.3-2.7,4.9l-15.9,4.6C919.9,847.2,919.5,847.3,919.2,847.3z"/>
+					</g>
+					<polyline class="st1" points="872.5,836.9 985.2,804.6 985.2,836.9 872.5,869.3 					"/>
+				</g>
+				<g>
+					<text transform="matrix(1.0085 0 0 1 969.4008 838.2935)" class="st3 st4 st5"></text>
```
Comment (Agusx1211) [inline src/assets/404-tree.svg]: Not all systems can display the  character

image

File: src/assets/404-tree.svg [hunk 2]
```diff
@@ -0,0 +1,385 @@
...
+							<rect x="770.3" y="620.7" class="st3" width="26.7" height="2.9"/>
+							<rect x="710.9" y="620.7" class="st3" width="55.3" height="2.9"/>
+						</g>
+					</g>
+				</g>
+			</g>
+		</g>
+	</g>
+	<text transform="matrix(1.0104 0 0 1 670.6898 516.4946)" class="st6 st15 st16">MISSING</text>
+	<text transform="matrix(1.0104 0 0 1 648.5199 550.0952)" class="st6 st15 st16">CONTRACT</text>
```
Comment (Agusx1211) [inline src/assets/404-tree.svg]: Maybe "missing loan" is more accurate

File: src/app/app.module.ts [hunk 3]
```diff
@@ -61,6 +62,10 @@ const appRoutes: Routes = [
   { path: 'loan/:id', component: LoanDetailComponent },
   { path: 'address/:address', component: AddressComponent },
   { path: 'profile', component: ProfileComponent },
+  { path: '404', component: NotFoundModule },
+  { path: '**',  redirectTo: '/404' },
+  // { path: 'not-found', loadChildren: './not-found/not-found.module#NotFoundModule'},
+  // { path: '**',  redirectTo: '/not-found' },
```
Comment (Agusx1211) [inline src/app/app.module.ts]: Why are those lines commented? is for a future use case?

File: src/app/not-found/not-found.component.html [hunk 4]
```diff
@@ -0,0 +1,16 @@
+<div class="margin-spacing" [style.height.px]="winHeight">
+  <div class="txt-container">
+    <div class="title">this loan does not exist</div>
+    <div class="miscellany"></div>
+    <div class="subtitle">Oooops, something went wrong</div>
+  </div>
+  <div class="img-container"></div>
+  <div id="clouds">
```
Comment (Agusx1211) [inline src/app/not-found/not-found.component.html]: The clouds don't re-appear outside off the screen

File: src/assets/404-tree.svg [hunk 5]
```diff
@@ -0,0 +1,386 @@
...
+							<rect x="770.3" y="620.7" class="st4" width="26.7" height="2.9"/>
+							<rect x="710.9" y="620.7" class="st4" width="55.3" height="2.9"/>
+						</g>
+					</g>
+				</g>
+			</g>
+		</g>
+	</g>
+	<text transform="matrix(1.0104 0 0 1 670.6895 516.4941)" class="st7 st16 st17">MISSING</text>
+	<text transform="matrix(1.0104 0 0 1 648.5195 550.0957)" class="st7 st16 st17">CONTRACT</text>
```
Comment (Agusx1211) [inline src/assets/404-tree.svg]: Maybe "missing loan" is more correct

File: src/assets/404-tree.svg [hunk 6]
```diff
@@ -0,0 +1,386 @@
...
+							c0.6,2.1-0.6,4.3-2.7,4.9l-16.9,4.8C957.7,835.1,957.3,835.1,956.9,835.1z"/>
+					</g>
+					<g>
+						<path class="st2" d="M919.2,847.3c-1.7,0-3.3-1.1-3.8-2.9c-0.6-2.1,0.6-4.3,2.7-4.9l15.9-4.6c2.1-0.6,4.3,0.6,4.9,2.7
+							c0.6,2.1-0.6,4.3-2.7,4.9l-15.9,4.6C919.9,847.2,919.5,847.3,919.2,847.3z"/>
+					</g>
+					<polyline class="st1" points="872.5,836.9 985.2,804.6 985.2,836.9 872.5,869.3 					"/>
+				</g>
+				<g>
+					<text transform="matrix(1.0085 0 0 1 969.4004 838.2939)" class="st4 st5 st6"></text>
```
Comment (Agusx1211) [inline src/assets/404-tree.svg]: Not all devices are able to display the  character

image

**Comment by ignacioperezroca** [comment]
### 404 - Not Found
-Cloud Animation
-"Missing Loan" Text
-Crossbrowsing --> SVG Compat

File: src/app/not-found/not-found.component.html (hunk 4 above)
Comment (ignacioperezroca) [reply src/app/not-found/not-found.component.html]: Done

File: src/assets/404-tree.svg (hunk 5 above)
Comment (ignacioperezroca) [reply src/assets/404-tree.svg]: Done

File: src/assets/404-tree.svg (hunk 6 above)
Comment (ignacioperezroca) [reply src/assets/404-tree.svg]: Done

**Pull request merged**